pkg/controllers/unsupported/unsupported_controller.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2024.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package unsupported
+
+import (
+	"context"
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/runtime"
+
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/builder"
+
+	configv1 "github.com/openshift/api/config/v1"
+	"github.com/openshift/cluster-capi-operator/pkg/controllers"
+	"github.com/openshift/cluster-capi-operator/pkg/operatorstatus"
+)
+
+// CapiUnsupportedController on unsupported platforms watches and keeps the cluster-api ClusterObject up to date.
+type CapiUnsupportedController struct {
+	operatorstatus.ClusterOperatorStatusClient
+	Scheme *runtime.Scheme
+}
+
+// Reconcile reconciles the cluster-api ClusterOperator object.
+func (r *CapiUnsupportedController) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
+	log := ctrl.LoggerFrom(ctx).WithName("CapiUnsupportedController")
+	log.Info(fmt.Sprintf("Reconciling %q ClusterObject", controllers.ClusterOperatorName))
+
+	if err := r.ClusterOperatorStatusClient.SetStatusAvailable(ctx); err != nil {
```

Comment:
I assume there's no way to set the status available with a message saying that we are running on an unsupported platform?

Reply:
We could potentially use the `message` field in the `type: Available` condition? 🤔 
Not sure if that would break other assuptions?
```
            "status": {
                "conditions": [
                    {
                        "lastTransitionTime": "2024-01-26T13:27:00Z",
                        "message": "Cluster CAPI Operator is available at 4.16.0-0.ci.test-2024-01-26-124620-ci-op-8kd6z0s4-latest",
                        "reason": "AsExpected",
                        "status": "True",
                        "type": "Available"
                    },
                    {
                        "lastTransitionTime": "2024-01-26T13:27:00Z",
                        "reason": "AsExpected",
                        "status": "False",
                        "type": "Progressing"
                    },
                    {
                        "lastTransitionTime": "2024-01-26T13:27:00Z",
                        "reason": "AsExpected",
                        "status": "False",
                        "type": "Degraded"
                    },
                    {
                        "lastTransitionTime": "2024-01-26T13:27:00Z",
                        "reason": "AsExpected",
                        "status": "True",
                        "type": "Upgradeable"
                    }
```

Reply:
I think message in available is the correct place, if you have concerns though, check with the OTA team

Reply:
I checked with them and they said it is the intended usage. I've update the PR to use it in the unsupported case to notify the user about it 👍